### PR TITLE
buffs TACTICAL medkits

### PR DESCRIPTION
**Pull request by jughu**
Tactical medkit burn/brute patches are switched out with synthflesh and 1 unit of inacusiate that will remove all ear damage (if you hit yourself with a bomb you dumb flukey!) only 2 per medkit as it replaced the burn/brute 

:cl: Hyena
tweak: Removes burn/brute patch from tactical medkits and replaces it with synthflesh
/:cl:

[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:) Tactical medkits should be tactical handling both damages at the same time or you get a tactical disadventage

### CODE REVIEW

**Comment by kevinz000** [comment]
No

File: code/modules/reagents/reagent_containers/patch.dm [hunk 1]
```diff
@@ -28,4 +28,9 @@
...
 	desc = "Helps with burn injuries."
 	list_reagents = list("silver_sulfadiazine" = 20)
-	icon_state = "bandaid_burn"
\ No newline at end of file
+	icon_state = "bandaid_burn"
+	
+/obj/item/reagent_containers/pill/patch/synth_flesh
+	name = "synthflesh patch"
+	desc = "Helps with burn and brute injuries."
+	list_reagents = list("synthflesh" = 20, "inacusiate" = 1,)
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/patch.dm]: did you test this

**Comment by jughu** [comment]
thank you for your informative feedback kevin

**Comment by GunHog** [comment]
Where do these items spawn?

File: code/modules/reagents/reagent_containers/patch.dm [hunk 2]
```diff
@@ -28,4 +28,9 @@
...
 	desc = "Helps with burn injuries."
 	list_reagents = list("silver_sulfadiazine" = 20)
-	icon_state = "bandaid_burn"
\ No newline at end of file
+	icon_state = "bandaid_burn"
+	
+/obj/item/weapon/reagent_containers/pill/patch/synth_flesh
+	name = "synthflesh patch"
+	desc = "Helps with burn and brute injuries."
+	list_reagents = list("synthflesh" = 20, "inacusiate" = 1,)
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/patch.dm]: did you test this

**Comment by jughu** [comment]
@GunHog its the nuke ops tactical medkit that costs tc

**Comment by Iamgoofball** [comment]
did you test this

**Comment by PKPenguin321** [comment]
>tactical

triggered

**Comment by jughu** [comment]
reeee test for what goof its litteraly 2 chems

**Comment by Iamgoofball** [comment]
if you tested you would notice the problem

**Comment by GunHog** [comment]
You need to test **ALL** your changes. You do not know what you screwed up until you _test_. The Github Online editor should be banned.

**Comment by KorPhaeron** [comment]
I saw "buff" and "tactical" in the PR title and my eyes glazed over and I really want to close this but I need to actually read it first to do so, what do I do gamers

**Comment by GunHog** [comment]
Make him test it first!

**Comment by PKPenguin321** [comment]
tactical must die

**Comment by ShadowDeath6** [comment]
tacticool medkit that has a single patch with 1000u cyanide please

**Comment by Repukan** [comment]
Never forget Oxford

**Comment by Haevacht** [comment]
You close it Kor.

Low effort shit from Jughu, and untested. This is pathetic.

**Comment by swindly** [comment]
A styptic powder patch and a silver sulfadiazine patch heal a total of 120 brute damage and 120 burn damage. Two synthflesh patches heal a total of 50 brute damage and 50 burn damage.

**Comment by jughu** [comment]
@swindly yeah but the two patches costs time to patch seperatly while s synthflesh combines the affects and plus fixes your hearing

**Comment by optimumtact** [comment]
technically this reduces the number of tactics, since now you only have one thing to apply rather than two.

**Comment by jughu** [comment]
Fuck oranges is right how could i been so blind

**Comment by lzimann** [comment]
I feel like this is more of a nerf than a buff, plus you add a random chem to synflesh because? And this seems disliked aswell so I'll just close.

**Comment by jughu** [comment]
@lzimann that random chem instantly fixes all your ear damage

**Pull request closed without merging**